exodus_lambda/functions/origin_request.py

```diff
@@ -172,23 +173,41 @@ def resolve_aliases(self, uri):
     def handle_cookie_request(self, event):
         now = datetime.utcnow()
         ttl = timedelta(minutes=self.conf.get("cookie_ttl", 720))
-        domain = event["Records"][0]["cf"]["config"]["distributionDomainName"]
-        uri = event["Records"][0]["cf"]["request"]["uri"]
+        record = event["Records"][0]["cf"]
+        domain = record["config"]["distributionDomainName"]
+        uri = record["request"]["uri"]
+        querystring_dict = parse_qs(record["request"]["querystring"])
+        policy = querystring_dict.get("Policy", [None])[0]
```

Comment:
I don't think these are going to work.  Per https://github.com/release-engineering/exodus-gw/pull/606#discussion_r1359954369 I believe we'll need to have three policies in play here.  If this code here works at all, it's currently picking the policy for `/_/cookie` which is not going to be reusable for any other path, so putting it into a cookie isn't helpful.

I would expect here that we need to pick out the policies for the cookie from a new query string element and not Policy/Signature which was already used to enable access to `/_/cookie`.